app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -385,15 +385,18 @@ private object TelemetryHomeTileUniqueClickPerSessionCounter {
     }
 }
 
-/** Tracks the name of the remote control used. Only the first remote used per session is reported. */
+/** Tracks the name of the remote control used. Only the first remote used per session is reported.
+ * If the remote name returns as null, we save as "null". This way, we can also determine how many
+ * users use remotes that are not recognized. */
 @UiThread // We get-and-set over SharedPreferences so we need resource protection.
 private object TelemetryRemoteControlTracker {
 
     fun saveRemoteControlInformation(context: Context, keyEvent: KeyEvent) {
         Assert.isUiThread()
         if (!TelemetryHolder.get().configuration.isCollectionEnabled) { return }
 
-        val remoteName = InputDevice.getDevice(keyEvent.deviceId).name
+        val remoteInfo = InputDevice.getDevice(keyEvent.deviceId)
+        val remoteName = if (remoteInfo != null) remoteInfo.name else "null"
```

Comment:
We could simplify this down to:
`val remoteName = InputDevice.getDevice(keyEvent.deviceId)?.name ?: "null"`